### DISCUSSION TITLE
Where can I find the saves to delete?

### DISCUSSION THREAD

**Discussion by Lenancha** [JordanSantiagoYT/FNF-JS-Engine#735] [Q&A] [answered]
I just want to know what folder it is in. Because I feel like I messed up something on my end.

**Comment by moxie-coder** [comment, marked as answer]
go into app data and look for JordanSantiago, that’s where the game saves are

**Reply by Lenancha** [reply]
Thank you!

**Reply by moxie-coder** [reply]
np
